src/counsellingTeam/models.py

```diff
@@ -32,3 +32,14 @@ class CsFaq(models.Model):
     category = models.CharField(max_length=15, choices=CATEGORY_CHOICES)
     question = models.TextField()
     answer = RichTextUploadingField(blank=True, null=True)
+
+
+USER = 1
```

Comment:
@vineethvanga18 The default user right now is the User with UserID 1? Is that correct?

Reply:
Yeah so we are using the same default here for guide and the user

Reply:
shouldn' we start with a centre (not a user but rather a point). because User with UserID 1 may not be the first person of the institute, it will be the first person in the DB.

Reply:
I thought we can change it afterwards. What we can do is create an id with username "initial" or something and use this id as the default one

Reply:
We should not have code depend on the DB, because then the installation process is longer, can you figure out another way?